src/xdp/apikernel.c

```diff
@@ -483,7 +484,7 @@ XdpApiKernelStart(
         Characteristics->ProviderRegistrationInstance.Size = sizeof(NPI_REGISTRATION_INSTANCE);
         Characteristics->ProviderRegistrationInstance.NpiId = &NPI_XDPAPI_INTERFACE_ID;
         Characteristics->ProviderRegistrationInstance.ModuleId = &NPI_XDP_MODULEID;
-        Characteristics->ProviderRegistrationInstance.Number = XDP_API_VERSION_1;
+        Characteristics->ProviderRegistrationInstance.Number = XDP_API_VERSION_LATEST;
```

Comment:
how does this work with multiple providers? xdp needs to be able to support multiple API versions simulatenously